core-to-plc/test/list.plc.golden

```diff
@@ -0,0 +1,9 @@
+Error: Error during conversion: Recursion while converting types
+Context: Converting type: [a]
+Context: Converting data constructor: :
```

Comment:
Maybe wrap operators in `()`? `constructor: :` looks rather ugly.

Reply:
This is just GHC's pretty-printing - I'd rather not mess with it if I can get away with it.

Reply:
Ah, didn't know this is due to GHC.